### DISCUSSION TITLE
Need to customize the sidebar titles

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#910] [Q&A]
For SEO purposes, I need to customize the sidebar names instead of using the page title. How can I achieve that?
I am using "fumadocs-core": `13.4.9` and "fumadocs-mdx": `10.0.2`

**Comment by fuma-nama** [comment]
You should add a custom field to frontmatter via source config and use it instead of the original title field in your page's metadata

**Reply by Ali-Hussein-dev** [reply]
Can you guide on how I can I add it to source config file?

**Reply by fuma-nama** [reply]
https://fumadocs.vercel.app/docs/mdx/configuration#extend-schema

**Comment by Gitstar-OC** [comment]
Is the issue resolved ?

**Reply by Ali-Hussein-dev** [reply]
not yet, couldn't figure it out how

**Reply by Gitstar-OC** [reply]
Can you pelase elaborate what do you mean by you want different name. Means you are saying that you don't want to use the page title in the mdx file nor the description but just want a name on the side bar? 

If this is what you mean please reply I have a solution for that

**Reply by Ali-Hussein-dev** [reply]
want to keep the title in mdx file while having different name in the sidebar.
![image](https://github.com/user-attachments/assets/b20a1b12-da65-4691-81c5-63115b069aab)

![image](https://github.com/user-attachments/assets/6332743a-d0f0-4464-a022-bf7e9a1cf32a)

**Reply by Gitstar-OC** [reply]
ok let me check.

**Reply by Gitstar-OC** [reply]
There is one possible option for this

What you can do is hide the title and description using css, which will be passed as metadata if you want, you can change that later, and for the heading of the page you can use #. 

```
---
title: Templates   // Hide this using CSS 
description: Curated premium and open source Next.js Starters. 
---
 
# Nextjs Templates
```

**Reply by fuma-nama** [reply]
I don't get it, simply changing your `page.mdx` can do the same thing much easier, and you should add another field to frontmatter for this.

**Reply by Gitstar-OC** [reply]
Like I wanted to add the 
```
level: <kbd> Level of the page <kbd> 
``` 

as custom component at the top of the page. I think he just needs this for one file.
